src/state/slices/stakingDataSlice/selectors.ts

```diff
@@ -1,36 +1,42 @@
 import { createSelector } from '@reduxjs/toolkit'
 import { CAIP10, CAIP19 } from '@shapeshiftoss/caip'
 import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import { chainAdapters } from '@shapeshiftoss/types'
 import { ValidatorReward } from '@shapeshiftoss/types/dist/chain-adapters/cosmos'
+import BigNumber from 'bignumber.js'
+import get from 'lodash/get'
+import memoize from 'lodash/memoize'
 import { ReduxState } from 'state/reducer'
 import { createDeepEqualOutputSelector } from 'state/selector-utils'
 
-export const ASSET_ID_TO_DENOM: { [k: CAIP19]: string } = {
-  'cosmos:cosmoshub-4/slip44:118': 'uatom'
+export type ActiveStakingOpportunity = {
+  address: string
+  moniker: string
+  apr: string
+  cryptoAmount?: string
+  rewards?: string
 }
 
-export const DENOM_TO_ASSET_ID: { [k: string]: CAIP19 } = {
-  uatom: 'cosmos:cosmoshub-4/slip44:118'
+export type amountByValidatorAddressType = {
+  [k: string]: string
```

Comment:
can we please comment what string and string are here? i assume validator address and staked ATOM in base units?

Reply:
Why not declare 2 types?

`export type ValidatorAddress = string`
`export type StakedAmount = string` 

It would be clearer for us to know, instead of just commenting here